src/helpers/color.ts

```diff
@@ -319,3 +319,7 @@ export function isSameColor(color1: Color, color2: Color, tolerance: number = 0)
   );
   return diff <= tolerance;
 }
+
+export function changeColorAlpha(color: Color, alpha: number): string {
```

Comment:
It's more usual to use get/set in names
```suggestion
export function setColorAlpha(color: Color, alpha: number): string {
```